Restore.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+set NuGetExe="%~dp0NuGet.exe"
+
+echo Restoring packages: Toolsets
+call %NugetExe% restore -verbosity quiet "%~dp0build\ToolsetPackages\project.json" -configfile "%~dp0nuget.config"
```

Comment:
I filed a bug against NuGet team; https://github.com/NuGet/Home/issues/1648. 

I can go back and forth on this, but I prefer quiet for now as it leads to a better experience.